cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/DiffLogic.java

```diff
@@ -69,7 +69,6 @@ public String createDiffOutput(Bean liveConfig, Bean desiredConfig) {
     private String doCreateDiffOutput(Bean liveConfig, Bean desiredConfig) {
         Differ differ = new Differ();
         differ.ignoreSpecBeanMapChange();
-        differ.ignoreRemovedObjects();
```

Comment:
Should we also delete the entire `ignoreRemovedObjects()` method in the `Differ` class?

Reply:
I think it wouldn't hurt to keep for now